store/modules/generator/generator.helpers.ts

```diff
@@ -60,18 +60,25 @@ export const helpers = {
         let additionnalPath = '';
         // Images are not directly stored at the root level
         if (pathsNumber > 3) {
```

Comment:
What is pathsNumber here exactly? Or maybe a better question, what is this check for?

Reply:
Sometime this function takes a "baseUrl" argument which is a manifest url ("https://aero.ms/manifest.json"). This url is splatted to get the base url (https://aero.ms) out of it.